CONTRIBUTING.md

```diff
@@ -33,7 +33,7 @@ outlined on that page and do not file a public issue.
 
 Same as in [README](README.md) with the exception of:
 ```
-python setup.py develop
+pip install -e .
```

Comment:
Does this now replace the two-step approach `python setup.py [mode]` + `pip install -r [mode]-requirements`?

Should we consider sticking with one flavor in our documentation (in README.md it still tells the users to do `python setup.py install`)?

Reply:
strange, i meant to change it in requirements.txt and not in contributing but maybe i wasnt paying attention